.gitignore

```diff
@@ -0,0 +1,3 @@
+composer.lock
+vendor
+phpunit.xml
```

Comment:
Okay.

What if we made changes to `phpunit.xml` file in our future released. This will not let this PHPUnit file to be uploaded because it's been excluded.

Correct me if I'm wrong. :)

We will also integrate it with travis, in near future.